src/python/WMCore/Services/Service.py

```diff
@@ -103,6 +103,10 @@ def __init__(self, cfg_dict = None):
         if not cfg_dict['endpoint'].endswith('/'):
             cfg_dict['endpoint'] = cfg_dict['endpoint'].strip() + '/'
 
+        # setup port 8443 for cmsweb services
+        if cfg_dict['endpoint'].startswith("https://cmsweb"):
```

Comment:
how about cmsweb-testbed is it using the same port?

Reply:
Yes, it has that port enabled too